tests/integration/test_content_manifest.py

```diff
@@ -19,3 +19,43 @@ def test_invalid_content_manifest_request(test_env):
         client.fetch_content_manifest(request_id=0)
     assert e.value.response.status_code == 404
     assert e.value.response.json() == {"error": "The requested resource was not found"}
+
+
+def test_valid_content_manifest_request(test_env, default_request):
+    """
+    Send a valid content-manifest request to the Cachito API.
+
+    Checks:
+    * Check that the response code is 200
+    * Check that the returned content contains a URL
+    pointing to a JSON schema for the content manifest file
+    * Check that the JSON schema is the same as in atomic-reactor repository
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+
+    initial_response = default_request.initial_response
+    content_manifest_response = client.fetch_content_manifest(initial_response.data["id"])
+    assert content_manifest_response.status == 200
+
+    response_data = content_manifest_response.data
+    assert_content_manifest_response(test_env, response_data)
+
+    icm_spec = response_data["metadata"]["icm_spec"]
+    assert_content_manifest_schema(test_env, icm_spec)
+
+
+def assert_content_manifest_response(test_env, response_data):
+    """Check the response data from Cachito content manifest."""
+    json_schema_url = test_env["content_manifest"]["expected_icm_spec"]
+    expected_response_data = {
+        "metadata": {"icm_version": 1, "icm_spec": json_schema_url, "image_layer_index": -1}
+    }
+    assert response_data == expected_response_data
+
+
+def assert_content_manifest_schema(test_env, icm_spec):
```

Comment:
@athos-ribeiro could you please comment here? Based on the acceptance criteria in CLOUDBLD-1534, this is what I understood was asked for but I don't find this a valuable test since the schema in the OSBS repo in master may change compared to the one linked from Cachito. I think it'd be more useful if Cachito got the JSON schema that was linked in the returned content manifest and then validate what was returned with that JSON schema.

What are your thoughts?

Reply:
That was an acceptance criteria for the first manual test, jut to make sure the value in the field was correctly pointing to the JSON schema.

Validating the returned schema here is a great idea.

Reply:
@mprahl @athos-ribeiro Thank you guys for pointing it. It is not clear to me yet, what kind of validation should I provide? That returned JSON schema is a correct JSON?

Reply:
@akhmanova I'm suggesting that you load the JSON schema from the URL in the content manifest returned by Cachito and validate that the content manifest itself is valid against the loaded JSON schema.